.github/labeler.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Labeler
+
+libraries:
+  - lib/**/*
+  - include/grass/*
+  - include/grass/**/*
+  - python/**/*
+
+module:
```

Comment:
I can't find this label either. Maybe worth adding it. (?)

We have vector, raster, temporal, and database.

Reply:
Just added support to generate vector, raster, temporal, and database labels too.